task_1/style.css

```
@@ -153,3 +151,66 @@
     font-size: 25px;
     font-weight: bolder;
 }
+
+.menu-icon{
+    width: 50px;
+    height: 50px;
+    cursor: pointer;
+    display: none;
+    margin-top: 20px;
+}
+
+.menu-item{
+    margin: 20px;
+}
+
+.menu-item:hover{
+    color: rgb(59, 74, 102);
+    cursor: pointer;
+} 
+
+@media screen and (max-width: 1200px) {
+    .hero-image {
+        display: none;
+    }
+}
+
+@media screen and (max-width: 700px) {
+    .brown {
+        display: none;
+    }
+}
+
+@media screen and (max-width: 500px) {
+    .blue {
+        display: none;
+    }
+}
+
+@media screen and (max-width: 900px) {
+    .view-more {
+        display: none;
+    }
+    .navbar {
+        flex-direction: column;
+        visibility: hidden;
+        justify-content: space-between;
+        background-color:rgb(167, 184, 216);
+        z-index: 1;
+        margin-top: 370px;
+        width: 100%;
+        justify-content: flex-end;
+    }
+    .header-item{
+        margin: 10px;
+    }
+    .menu-icon{
+        display: inline;
+    }
+    .sub-title-p{
+        width: auto;
+    }
+    .header{
```

line space between class 
Apply it in the whole file
```suggestion

    .header{
```